delivery_carrier_label_default/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Camptocamp SA
+# Copyright 2021 Sunflower IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Default label for carrier labels",
+    "summary": "This module defines a basic label to print "
+    "when no specific carrier is selected.",
+    "version": "14.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Delivery",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Camptocamp, Sunflower IT, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "stock",
```

Comment:
redundant, it is already installed via the dependency chain below:

base_delivery_carrier_label -> delivery_carrier_info -> delivery -> sale_stock ->stock_account -> [ stock ]